chrome/content/zotero/zoteroPane.xul

```diff
@@ -106,7 +107,19 @@
 					<toolbarbutton id="zotero-tb-library-add-menu" class="zotero-tb-button" tooltiptext="&zotero.toolbar.newLibrary.label;" type="menu">
 						<menupopup id="zotero-tb-library-add-popup">
 							<menuitem id="zotero-tb-group-add" label="&zotero.toolbar.newGroup;" oncommand="ZoteroPane_Local.newGroup()"/>
-							<menuitem id="zotero-tb-feed-add" label="&zotero.toolbar.newFeed.label;" command="cmd_zotero_newFeed"/>
+							<menu label="&zotero.toolbar.feeds.new;">
+								<menupopup id="zotero-tb-feed-add-more">
```

Comment:
Should put an id — `zotero-tb-feed-add-menu` — on the menu instead. (Will become clear why shortly.)